_posts/2022-05-01-release-table-store-0.1.0.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title:  "Apache Flink Table Store 0.1.0 Release Announcement"
+subtitle: "Unified streaming and batch store for building dynamic tables on Apache Flink."
+date: 2022-05-01T08:00:00.000Z
+categories: news
+authors:
+- Jingsong Lee:
+  name: "Jingsong Lee"
+
+---
+
+The Apache Flink Community is pleased to announce the preview release of the
```

Comment:
'Community' better be lowercased here.

Ref
![image](https://user-images.githubusercontent.com/55568005/166945854-91d287d0-39d2-4b75-a21b-ac699331ee69.png)